### DISCUSSION TITLE
Support with merge two companies using Google Workspace for SSO login via google to draw.io

### DISCUSSION THREAD

**Discussion by bkaluzny-xtm** [jgraph/drawio#4381] [General]
We are managing the merger of two companies, both currently using Google Workspace for SSO. We are planning to perform a two-step workspace integration and we identified a possible problem with maintaining user identity in your platform during these changes. The first step is to merge workspace A to workspace B, which means OAuth2 issuer+subject claims will change, but the email addresses will remain the same. The second step is to change the primary domain of the merged workspace, so issuer+subject claims will not change, but the email will (domain part only). What data is used for your SSO? Is it issuer+subject, or is it an email address? How can draw.io support us with the above merger in order to retain a continuous association of your account with actual people in our companies?

**Comment by davidjgraph** [comment]
There's no account in draw.io, users just open diagrams from drive.google.com into the application and save it back to Drive. It makes no difference what the email address is.
